view/concept-info.twig

```diff
@@ -1,8 +1,9 @@
 
 {% extends "light.twig" %}
 {% set concept_label = '' %}{% for concept in search_results %}{% set concept_label = concept.label %}{% endfor %}
-{% block title %}: {{ vocab.shortName }}{% if search_results|length == 1 %}: {{ concept_label }}{% endif %}
-{% endblock %}
+{% block title %}{{ concept_label }} - {{ vocab.shortName }} - {{ GlobalConfig.serviceName }}{% endblock %}
+{% block description %}{% set term = concept_label %}{% set vocabObj = vocab %}{% set vocab = vocab.title(request.contentLang) %}{% trans %}Concept {{ term }} in vocabulary {{ vocab }}{% endtrans %}{% set vocab = vocabObj %}{% endblock %}
```

Comment:
The minor quip is that we have only the translation strings for "Concept X" in different languages, whereas it look a bit weird in cases where the data is instances or classification:
 - Skosmos2/ykl/en/page/2
   - "Concept RELIGION in vocabulary PLC - Finnish Public Libraries Classification System"
   - Could be currently generated as "YKL Class RELIGION ..."
 - Skosmos2/finaf/en/page/000179638
   - "Concept A. Aallon Rytmiorkesteri in vocabulary KANTO - Kansalliset toimijatiedot"
   - Could be currently merely generated as "Yhteisö A. Aallon Rytmiorkesteri ..." as there's no translation for the concept type in english in the data
   
  One way would be to omit the word "Concept" but it's a minor thing.

Reply:
Good point! This was part of the speccing done earlier. I guess we didn't consider classifications back then.